schema-registry-examples/README.md

```diff
@@ -0,0 +1,54 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
```

Comment:
Please, add a title here (e.g., `Schema Registry Samples` or similar, as in the other folders of this repo)